### PR TITLE
website/docs: Add Kubernetes Bootstrap Instructions

### PR DESCRIPTION
**Pull request by Zapfmeister**
## Details
Add instructions on the bootstrap parameters for Kubernetes Deployment, as per:
https://github.com/goauthentik/authentik/issues/3080

---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

File: website/docs/installation/automated-install.md [hunk 1]
```diff
@@ -23,3 +23,20 @@ Requires authentik 2023.3
...
+  bootstrap_password: test
+```
+
+To store it in a secret, use:
+```
+envFrom
+ - secretRef:
+     name: some-secret
+```
+where some-secret would contain the environment variables as in the documentation.
```
Comment (tanberry) [inline website/docs/installation/automated-install.md:42]:
```suggestion
where _some-secret_ contains the environment variables as in the documentation above.
```

File: website/docs/installation/automated-install.md [hunk 2]
```diff
@@ -23,3 +23,20 @@ Requires authentik 2023.3
...
+## Kubernetes
+
+Set the `akadmin`user password and token:
+```
+authentik:
+  bootstrap_token: test
+  bootstrap_password: test
+```
+
+To store it in a secret, use:
```
Comment (tanberry) [inline website/docs/installation/automated-install.md:36]:
```suggestion
To store the password and token in a secret, use:
```

File: website/docs/installation/automated-install.md [hunk 3]
```diff
@@ -23,3 +23,20 @@ Requires authentik 2023.3
 :::
 
 Set the email address for the default `akadmin` user.
+
+## Kubernetes
+
+Set the `akadmin`user password and token:
```
Comment (tanberry) [inline website/docs/installation/automated-install.md:29]: Can you state exactly where they set this? I am assuming in their `.env` file, but let's make it explicit in the instructions. ;-)

File: website/docs/installation/automated-install.md [hunk 4]
```diff
@@ -23,3 +23,20 @@ Requires authentik 2023.3
...
+authentik:
+  bootstrap_token: test
+  bootstrap_password: test
+```
+
+To store it in a secret, use:
+```
+envFrom
+ - secretRef:
+     name: some-secret
```
Comment (tanberry) [inline website/docs/installation/automated-install.md:40]:
```suggestion
     name: _some-secret_
```

File: website/docs/installation/automated-install.md (hunk 1 above)
Comment (tanberry) [reply website/docs/installation/automated-install.md:42]: Hi @Zapfmeister first off all, thank you very much for adding this to our docs; we definitely need more How To/procedural docs. This is a great addition. 

For the last sentence (line 42), can you explain a bit more what that means? Maybe give an example with the exact syntax? Which one of the three variables listed above in the docs would one use for the `some-secret`? Any of them? 

Thanks, I am still learning authentik myself, and I want to make the docs easy for newbies, too.

**Review by tanberry — CHANGES_REQUESTED** [review]
Thanks so much! A few edits and questions...

**Comment by tanberry** [comment]
Please also be sure to run `make website` or `npm run prettier` before pushing your changes; the linter is breaking on the build checks. (Don't worry about the other build fails.)

File: website/docs/installation/automated-install.md [hunk 5]
```diff
@@ -23,3 +23,20 @@ Requires authentik 2023.3
...
+Set the `akadmin`user password and token:
+```
+authentik:
+  bootstrap_token: test
+  bootstrap_password: test
+```
+
+To store it in a secret, use:
+```
+envFrom
```
Comment (rissson) [inline website/docs/installation/automated-install.md:38]: This should be under a `global:` key.

Also:
```suggestion
envFrom:
```

**Comment by tanberry** [comment]
Hi @Zapfmeister are you still interested in working on this PR? Would be great if so, just needs a few tweaks then let's merge.

**Comment by Zapfmeister** [comment]
> Hi @Zapfmeister are you still interested in working on this PR? Would be great if so, just needs a few tweaks then let's merge.

@tanberry i think it should be fine this way. Im not sure about the secret reference as per https://github.com/goauthentik/authentik/issues/3080#issuecomment-1153174175, as i never used it.

**Comment by rissson** [comment]
> > Hi @Zapfmeister are you still interested in working on this PR? Would be great if so, just needs a few tweaks then let's merge.
> 
> @tanberry i think it should be fine this way. Im not sure about the secret reference as per [#3080 (comment)](https://github.com/goauthentik/authentik/issues/3080#issuecomment-1153174175), as i never used it.

It's in the Helm values

File: website/docs/installation/automated-install.md (hunk 3 above)
Comment (rissson) [reply website/docs/installation/automated-install.md:29]:
```suggestion
In the Helm values, set the `akadmin`user password and token:
```

**Review by tanberry — APPROVED** [review]
Thanks @Zapfmeister and @rissson ! I believe all requested changes have been made, let's merge!

**Pull request merged**